src/EditorFeatures/Core/project.json

```diff
@@ -1,19 +1,19 @@
 {
   "dependencies": {
     "System.Collections.Immutable": "1.3.1",
-    "Microsoft.VisualStudio.InteractiveWindow": "2.0.0-beta5-60820-04",
-    "Microsoft.VisualStudio.Text.Internal": "14.3.25407-alpha",
-    "Microsoft.VisualStudio.Language.CallHierarchy": "14.3.25407-alpha",
-    "Microsoft.VisualStudio.Language.NavigateTo.Interfaces": "14.3.25407-alpha",
-    "Microsoft.VisualStudio.Language.StandardClassification": "14.3.25407",
-    "Microsoft.VisualStudio.Language.Intellisense": "14.3.25407",
-    "Microsoft.VisualStudio.Imaging": "14.3.25407",
+    "Microsoft.VisualStudio.InteractiveWindow": "2.0.0-rc3-61304-01",
+    "Microsoft.VisualStudio.Text.Internal": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Language.CallHierarchy": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Language.NavigateTo.Interfaces": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Language.StandardClassification": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Language.Intellisense": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Imaging": "15.0.26014-alpha",
     "Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime": "14.3.25407",
```

Comment:
@jasonmalinowski it doesn't affect our ability to run.  It's still unclear what the right thing to do actually is.  It' an interp assembly, target 14.0, whose version is changing.  That is confusing.  I'm going to dig a bit deeper into it and if it needs to change going to submit another issue.